PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Brief description of changes:
+
+
+## Relevant link(s):
+Please include `closes #{issue number}` if appropriate
+
+
+#### Optional
+- Discuss architecture changes or design decisions
+- [Add yourself the contributors list](https://github.com/cannawen/metric_units_reddit_bot/blob/master/CONTRIBUTING.md#add-yourself-to-the-contributors-list)
```

Comment:
What do you think about adding a checklist for stuff? Also surround this in a comment saying "To be filled by a collaborator". It also serves as a nice reminder to the PR author IMO. Something like:

```
- [ ]  Adheres to the Style Guide
- [ ]  Behaviour is appropriately tested. If improvement, tests are adequately extended.
- [ ]  The number of reviews is appropriate for this type of PR
```
and stuff like this. So that in future, all changes approved by other than you follow the right way<sup>TM</sup> so you just have to merge after checking the ticks😛

Reply:
Ooh, check boxes for tasks like that would be nice! I'll look into adding those. Thank you for the feedback :D